docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -857,6 +857,32 @@ EXAMPLE: A more complex environment file:
    to get any bug fixes whilst still maintaining consistency of
    software environment.
 
+Starting from conda 4.14.0 release, multiple entries of ``pip:`` can be used for
```

Comment:
The version number 4.14.0 here is the next version of conda release (currently 4.13.0). This is arbitrary taken in the document. It should be updated before we merge this PR.